aiosmtpd/tests/test_handlers.py

```diff
@@ -1,9 +1,11 @@
+import asyncio
 import os
 import sys
 import unittest
 
 from aiosmtpd.controller import Controller
-from aiosmtpd.handlers import Debugging, Mailbox, Message, Proxy, Sink
+from aiosmtpd.handlers import Debugging, Mailbox, Message, Proxy, Sink, \
+    AsyncMessage
```

Comment:
Please use parentheses instead of backslashes.   Also, the imported names must be alphabetically sorted.  E.g.:

```
from aiosmtpd.handlers import (
    AsyncMessage, Debugging, Mailbox, Message, Proxy, Sink)
```